src/grid/submission/SubmissionGridBody.component.ts

```diff
@@ -22,9 +25,17 @@ export class SubmissionGridBodyComponent extends GridBodyComponent {
    */
   view(row: any, header: any) {
     const cellValue: any = get(row, header.key);
-    if (typeof header.component.getView === 'function') {
-      return header.component.getView(cellValue);
+    if (header.getView) {
+      return header.getView(cellValue);
```

Comment:
@vlad-shusterman a small enhancement suggestion here: it might be beneficial to also pass `header.component` as  second argument here in `getView` function. Somebody might want to use something from it, for ex. using `component.getView` method for their custom rendering logic.

Another small suggestion is that maybe it would be better to change `getView` function name in column and header definitions to something like `renderCell` which sounds semantically much more clear